components/brave_wallet_ui/page/screens/swap/hooks/useJupiter.ts

```diff
@@ -152,26 +153,27 @@ export function useJupiter(params: SwapParams) {
 
       let jupiterQuoteResponse
       try {
-        jupiterQuoteResponse = await swapService.getJupiterQuote({
-          chainId: selectedNetwork.chainId,
-          inputMint:
+        jupiterQuoteResponse = await swapService.getQuote({
+          fromAccountId: overriddenParams.fromAccount.accountId,
+          fromChainId: selectedNetwork.chainId,
+          fromToken:
             overriddenParams.fromToken.contractAddress ||
             WRAPPED_SOL_CONTRACT_ADDRESS,
-          outputMint:
-            overriddenParams.toToken.contractAddress ||
-            WRAPPED_SOL_CONTRACT_ADDRESS,
-          amount: !isFromAmountEmpty
+          fromAmount: !isFromAmountEmpty
```

Comment:
perf: flip the ternary to check `isFromAmountEmpty` instead of `!isFromAmountEmpty`